qgisfeedproject/qgisfeed/views.py

```diff
@@ -107,8 +110,72 @@ def get(self, request):
 def feeds_list(request):
     """
     List of feeds
-    This view will be updated later,
-    it is used to handle the login feature
-    for now
+    This view renders a paginated, sorted according
+    to the request parameters list of feeds
     """
-    return render(request, 'feeds/feeds_list.html')
\ No newline at end of file
+    form = FeedEntryFilterForm(request.GET)
+    feeds_entry = QgisFeedEntry.objects.all()
+
+    # Get filter parameters from the query string
+    if form.is_valid():
+        title = form.cleaned_data.get('title')
+        if title:
+            feeds_entry = feeds_entry.filter(title__icontains=title)
+            
+        author = form.cleaned_data.get('author')
+        if author:
+            feeds_entry = feeds_entry.filter(author__username__icontains=author)
+    
+        language_filter = form.cleaned_data.get('language_filter')
+        if language_filter:
+            feeds_entry = feeds_entry.filter(language_filter=language_filter)
+
+        publish_from = form.cleaned_data.get('publish_from')
+        if publish_from:
+            feeds_entry = feeds_entry.filter(publish_from__gt=publish_from)
+
+        publish_to = form.cleaned_data.get('publish_to')
+        if publish_to:
+            feeds_entry = feeds_entry.filter(publish_to__lt=publish_to)
+
+
+    # Get sorting parameters from the query string
+    sort_by = request.GET.get('sort_by', 'publish_from')
+    order = request.GET.get('order', 'desc')
+    current_order = order
+
+    if order == 'asc':
+        feeds_entry = feeds_entry.order_by(sort_by)
+        order = 'desc'
```

Comment:
It's bit confusing here, probably just use variable 'next_order'

Reply:
@dimasciput Tested ACK. You're right, I've fixed it.